lib/src/handler/json_wire/utils.dart

```diff
@@ -0,0 +1,143 @@
+import 'dart:convert';
+
+import 'package:webdriver/src/common/web_element.dart';
+
+import '../../common/exception.dart';
+import '../../common/request.dart';
+
+/// Magic constants -- identifiers indicating a value is an element.
+/// Source: https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol
+const String jsonWireElementStr = 'ELEMENT';
+
+dynamic parseJsonWireResponse(WebDriverResponse response,
```

Comment:
Return 'Map' instead of just 'dynamic'?

Reply:
Unfortunately, that's not always true. For example, the response for getting all current windows is actually a list.

See https://www.w3.org/TR/webdriver1/#get-window-handles and https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidwindow_handles

It can also be string etc.